lib/resources/http.rb

```diff
@@ -25,13 +25,14 @@ class Http < Inspec.resource(1)
     "
 
     # rubocop:disable ParameterLists
-    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil)
+    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil, ssl_verify: true)
```

Comment:
@chris-rock we need to transform this resource to support running HTTP checks on the remote node eventually, and I think it's unlikely we'll be able to use Faraday for that. Instead, we'll likely have to try to use curl, wget, python, etc. Having the user pass in something that's Faraday-specific doesn't make sense in that case, and it also requires the user to know more about the implementation of the resource itself. I'm all for using Faraday locally, though.

I suggest we leave this as is and continue to expose the tunables that our users need but can also be support simultaneously between local and remote targets depending on how we implement them.